drivers/enc28j60/enc28j60.c

```diff
@@ -330,7 +330,7 @@ static int nd_recv(netdev_t *netdev, void *buf, size_t max_len, void *info)
     next = (uint16_t)((head[1] << 8) | head[0]);
     size = (uint16_t)((head[3] << 8) | head[2]) - 4;  /* discard CRC */
 
-    DEBUG("[enc28j60] recv: size=%i next=%i buf=%p len=%d\n",
+    DEBUG("[enc28j60] recv: size=%i next=%i buf=%p len=%" PRIuSIZE "\n",
```

Comment:
If you feel like, you could change from `%i` to `%u` since `size` and `next` are unsigne, while you're at it anyway.

Reply:
Done. By the way, this is a fairly common problem in the codebase.